### ISSUE TITLE
Call for unified training script for multiple object detection models 

### ISSUE THREAD

**Issue by apple2373** [chainer/chainercv#626]
Similar to image classification examples (https://github.com/chainer/chainer/tree/master/examples ), it's nice if we can make a single training script that can switch multiple models in command line argument. Currently Faster RCNN and SSD have different scripts for training.  YOLO lacks any training script. I think we just need a single script that can specify the detection model as an command line argument.

**Comment by Hakuyume** [comment]
Faster R-CNN, SSD and YOLO use different data augmentation method and different learning rate policy from each other. Therefore, the unified training script will be very complicated and lack readability.
